multimodal/src/autogluon/multimodal/data/process_text.py

```diff
@@ -239,12 +239,10 @@ def build_one_token_sequence(
         Construct one token sequence based on multiple token sequences coming from different
         text columns in a multimodal pd.DataFrame. The token sequence length and the text segment
         id are upper bounded by "self.max_len" and "self.text_segment_num".
-
         Parameters
         ----------
         text_tokens
             One sample's text token sequences from different text columns in a multimodal pd.DataFrame.
-
```

Comment:
We'd better not remove the two \newlines. Without them, the docstring format changes.

Reply:
Done, thanks!